src/generators.h

```diff
@@ -160,6 +167,8 @@ std::shared_ptr<GeneratorParams> CreateGeneratorParams(const Model& model);
 std::shared_ptr<GeneratorParams> CreateGeneratorParams(const Config& config);  // For benchmarking purposes only
 std::unique_ptr<Generator> CreateGenerator(const Model& model, const GeneratorParams& params);
 
+void CopyThroughCpu(DeviceBuffer& dest, size_t begin_dest, DeviceBuffer& source, size_t begin_source, size_t size_in_bytes);
```

Comment:
A comment explaining what CopyThroughCpu means here would be helpful.

Reply:
I see the comment in the cpp. :)
Could we move it here?

Reply:
Yep, I put it in both places as it was small.